src/plugins/python/tiledbinding.py

```diff
@@ -331,6 +331,8 @@ def _decorate(obj, *args, **kwargs):
 cls_layer.add_method('asObjectGroup', retval('Tiled::ObjectGroup*',reference_existing_object=True), [])
 cls_layer.add_method('asImageLayer', retval('Tiled::ImageLayer*',reference_existing_object=True), [])
 cls_layer.add_method('asGroupLayer', retval('Tiled::GroupLayer*',reference_existing_object=True), [])
+cls_layer.add_method('offsetx', 'double', [])
+cls_layer.add_method('offsety', 'double', [])
```

Comment:
This change as such is insufficient. You'll need to actually run the `tiledbinding.py` script to update the `pythonbind.cpp` file.

In addition, the `Layer` class does not have `offsetx` and `offsety` methods. It has `QPointF offset() const;`. Probably what we need to add here is `cls_layer.add_method('offset', 'QPointF', [])`.

Reply:
I see. I thought the build process would regenerate that file for me :frowning_face: 
Please cancel this, I'll see if I can do better.

Reply:
I've pushed the necessary changes.